ERCS/erc-7786.md

```diff
@@ -103,6 +103,12 @@ This event signals that a would-be sender has requested a message to be sent.
 
 If `outboxId` is present, post-processing MAY be required to send the message through the cross-chain channel.
 
+#### Mandatory and optional attributes
+
+The `attributes` parameter MUST contain as its first element the value `read(true)` or `read(false)`. `read(true)` means the message can be read by the application on the destination gateway. `read(false)` means the message by pushed by the destination gateway on the application via `executeMessage`(see below).
+
+In the case `read(true)` is set as teh first element of the `attributes` parameters, the next element MUST be `messageId(val)` where val is a sequence of 32 bytes.
```

Comment:
- typo "teh" -> "the"
- didn't Francisco and CJ said they would want to add `msgId` as mandatory for all?

Reply:
> typo "teh" -> "the"

Thanks, fixed in 6ad57be0522f5bed7be627b1051335710d6b3376.

> didn't Francisco and CJ say they would want to add msgId as mandatory for all?

That is possible indeed.
cc @frangio  @cjcobb23

Reply:
`messageId` has been defined as a default attribute in 15c1f1b7dea821026f012451b6191e1c93093c5a.